### DISCUSSION TITLE
No ability to restore deleted skins, collections and scores

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#24073] [Q&A]
"soft deletion everywhere" is an advertised feature in the "what is lazer?" video but seems to be missing for some user data

![image](https://github.com/ppy/osu/assets/36758269/208dc773-dac7-4951-a46c-6fea814df504)

**Comment by peppy** [comment]
The correct path forward here is offering an "undo" button in the deletion notifications. This can already be done with how flexible notifications are (will require a new notification type though).

**Reply by Walavouchey** [reply]
to my understanding you can restore deleted data until you close the client, so i suppose this notification would need to be non-dismissible to match

**Reply by peppy** [reply]
Dismissible is fine. The idea is only to give users a last chance, not to "make it work until the client is closed".
